app/controllers/sign_up/completions_controller.rb

```diff
@@ -31,34 +27,29 @@ def update
 
     private
 
-    def handle_verified_attributes
-      update_verified_attributes
-      clear_verify_attributes_sessions
+    def verify_confirmed
+      redirect_to idv_url if current_user.decorate.identity_not_verified?
     end
 
-    def view_model
-      SignUpCompletionsShow.new(
-        ial2_requested: ial2?,
-        decorated_session: decorated_session,
-        current_user: current_user,
-        handoff: new_service_provider_attributes,
-        ialmax_requested: ialmax?,
-        consent_has_expired: consent_has_expired?,
-      )
+    def verify_needs_completions_screen
+      return_to_account unless needs_completions_screen?
     end
 
-    def verify_confirmed
-      redirect_to idv_url if current_user.decorate.identity_not_verified?
+    def completions_presenter
+      CompletionsPresenter.new(
+        current_user: current_user,
+        current_sp: current_sp,
+        decrypted_pii: pii,
+        requested_attributes: decorated_session.requested_attributes.map(&:to_sym),
+        ial2_requested: sp_session[:ial2] == true || sp_session[:ialmax] == true,
```

Comment:
QQ: why the `== true`?
```suggestion
        ial2_requested: sp_session[:ial2] || sp_session[:ialmax],
```

Reply:
I do not know, I was copying what was there before. Can clean it up though!